src/Server/RestRouterHandlers/TableRestRouterHandler.cpp

```diff
@@ -277,6 +290,19 @@ String TableRestRouterHandler::getCreationSQL(const Poco::JSON::Object::Ptr & pa
         create_segments.push_back(", shard=" + shard);
     }
 
+    std::vector<String> create_table_settings;
+    for (const auto & setting : CREATE_TABLE_SETTINGS)
+    {
+        if (hasQueryParameter(setting))
+        {
+            const String & setting_str = setting + "=" + getQueryParameter(setting);
+
+            create_segments.push_back(", " + setting_str);
+            create_table_settings.push_back(setting_str);
+        }
+    }
+    query_context->setQueryParameter("create_table_settings", boost::algorithm::join(create_table_settings, "&"));
```

Comment:
Only when `distributed_ddl=true`, we will need setup query parameter in context ? Actually, can we bindly setup all URL parameters in one shot : is it possible to get URL like `a=b&c=d&e=f...` and then setup `query_context->setQueryParameter("url_paramaters", url_params)` ? Then we don't need introduce different names for different url params ?